turbo.json

```diff
@@ -12,6 +12,10 @@
     },
     "storybook:run": {
       "cache": false
+    },
+    "storybook:build": {
+      "dependsOn": ["@webstudio-is/icons#build"],
```

Comment:
what is special about the icons package? is it because it's the monorepo package? if yes wouldn't it be easier to maintain if we used asterisk without specific package name in it?

Reply:
I'm resolving this one to proceed for now. We agreed to look into builds in a separate PR